src/modeler/AutoRest.Swagger/Validation/BodyPropertiesNamesCamelCase.cs

```diff
@@ -48,9 +48,21 @@ public class BodyPropertiesNamesCamelCase : TypedRule<Dictionary<string, Operati
         /// </summary>
         public override Category Severity => Category.Error;
 
-        ///// <summary>
-        ///// Validates whether property names are camelCase in body parameters.
-        ///// </summary>
+        /// <summary>
+        /// What kind of open api document type this rule should be applied to
+        /// </summary>
+        public override ServiceDefinitionDocumentType ServiceDefinitionDocumentType => ServiceDefinitionDocumentType.Default;
```

Comment:
Not really, communication over REST channels that rely on json and Json is usually camelCased.